content/concepts/introduction-to-webpack.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Introduction
+---
+
+*Webpack* is a module system for modern Javascript applications.
+
+In [modular programming](https://en.wikipedia.org/wiki/Modular_programming), developers break programs up into discrete chunks of functionality.
+This has a number of benefits.
+Each module has a smaller surface area for testing than a full program, making verification and bug-hunting easier.
+Well-written modules provide solid abstractions and encapsulation boundaries, so that each module has a coherent design and a clear purpose within the overall application.
+
+Node has supported modular programming almost since its inception.
+On the web, however, support for modules has been slow to arrive.
+Multiple tools exist that support modular Javascript on the web, with a variety of benefits and limitations.
+Webpack builds on lessons learned from these systems.
+
+In a Webpack application, each _module_ contains all of the code necessary to execute a single aspect of the application.
+Modules do not need to be self-contained.
+Code provided by one module can be reused in a second module: the first module _depends on_ the second module.
+
+In Webpack, dependencies can be expressed in many ways. A few examples are:
+
+<!-- * A module written in Javascript can depend on another module using the CommonJS `var module = require(modulePath)` function, or using AMD's `define([modulePath], function(module) {})` function.
+* A module written in ES6 can depend on another module using ES6's `import module from 'modulePath'` syntax.
+* A module written in CSS can depend on another module using CSS `@import` rules.
+ -->
+
+* A Javascript `require()` statement
+* An ECMAScript2015 `import` statement
+* An AMD `define` and `require` statement
+* An `@import` statement inside of a css/sass/less file.
+* An image url in a stylesheet or html file. 
+
+Any time one file depends on another, webpack treats this as a dependency, and therefore Webpack can load modules that contain non-code assets, such as images or web fonts, and can provide these modules as dependencies for your application.
+
+When Webpack processes your application, it starts from a list of modules defined on the command line or in its config file.
+Starting from these _entry points_, Webpack recursively builds a _dependency graph_ that includes every module your application needs, then packages all of those modules into a small number of _bundles_ - often, just one - to be loaded by the browser.
+Bundling your application is especially powerful for HTTP/1.1 clients, as it minimizes the number of times your app has to wait while the browser starts a new request.
+
+Webpack supports modules written in many languages, and allows developers to define new _loaders_ so that modules written in any language you like can be bundled with your application.
+The Webpack community has built loaders for a wide variety of popular languages and language processors, including:
```

Comment:
Ok, how about this:

1. `get-started.md` - Goal: get the first webpack build up and running, touch on a few basic concepts. Link to `concepts/index.md` (concept intro) at the end. Also to API and how to intros.
2. `concepts/index.md` - Goal: understand the basic concepts of webpack. This article could easily become that. Link to specific concepts.
3. `concepts/loaders.md` etc. - Goal: understand a specific concept.

There's not going to be a linear path as a lot depends on what the user knows already. But it would look something like that for a greenhorn. A more experience user would just skip these docs and dig deeper straight to how-tos and api docs.